src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -23,8 +23,17 @@ import {
   PrometheusRule
 } from '../app/shared/models/prometheus-alerts';
 
-export function configureTestBed(configuration: any) {
-  configureTestSuite(() => TestBed.configureTestingModule(configuration));
+export function configureTestBed(configuration: any, entryComponents?: any) {
+  configureTestSuite(() => {
+    if (entryComponents) {
```

Comment:
Will that be removed again after removing the entry components from the modules? Or does it makes sense to wait until they are removed?

Reply:
Jest currently doesn't work without entryComponents being declared.
So this would only be removed when Jest fixes this.
I'm thinking of using this to fix the PR that removes entryComponents from modules.

Reply:
Could you leave a comment with the given explanation in the code too? That would be great :)